proxyd/backend.go

```diff
@@ -931,6 +932,13 @@ func (bg *BackendGroup) ProcessMulticallResponses(ch chan *multicallTuple, ctx c
 			finalResp = resp
 			continue
 		}
+
+		// Assuming multicall doesn't support batch
+		if bg.multicallRPCErrorCheck && resp.RPCRes[0].IsError() {
```

Comment:
@jelias2 is this ok? Assuming multicall doesn't support batch

Reply:
Correct multi-call assumes single rpc request, no batches allowed